tests/queries/0_stateless/02115_write_buffers_finalize.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+${CLICKHOUSE_CLIENT} --query "DROP TABLE IF EXISTS file"
+${CLICKHOUSE_CLIENT} --query "CREATE TABLE file (x UInt64) ENGINE = File(Native, '${CLICKHOUSE_DATABASE}/data.xz')"
```

Comment:
Worth adding the same for all compression formats, like in https://github.com/ClickHouse/ClickHouse/blob/69cd19a3422b5dd7964656f7b1a89615a7812325/tests/queries/0_stateless/01059_storage_file_compression.sh#L9 ?

Reply:
The bug was reproduced only for `xz`, because it has quite long final flush in destructor, but let's add all compression formats, because thay all had flushing in destructors

Reply:
Oh, it's also reproduced with bz2, fine